app/webpack/users/edit/ducks/relationships.js

```diff
@@ -1,8 +1,6 @@
 import _ from "lodash";
 import inatjs from "inaturalistjs";
 
-import { fetchUserSettings } from "./user_settings";
-
```

Comment:
`user_settings.js` imported functions from this file which caused the circular dependency